source/voting.rst

```diff
@@ -0,0 +1,154 @@
+   unable to participate in the discussion for whatever reason. 
+
+#. If the steering committee cannot find a consensus, then they fall back to a
+   majority vote. 
+
+This is what we hereafter may refer to as “the decision making process”.
+
+.. note:: 
+   If a veto -1 vote or rejecting review is cast on a lazy consensus (e.g. on a
+   documentation change where the full decision making process is not required), 
+   the proposer/author can appeal to the
+   community and project members and the change can be approved or rejected using
+   the decision making procedure outlined above.
+
+.. important:: 
+   Project members are expected to attempt to resolve issues with
+   the contributor before casting a veto vote or rejecting review. 
+   Resolving issues here may be
+   proposing alternative implementations, alternative language, or a variant of
+   the solution that the project member feels is more workable. Project members
+   are expected to use veto votes and rejecting reviews sparingly. 
+
+Decisions (in addition to adding project members and Steering Committee
+membership) are made according to the following rules:
+
+Minor Documentation Changes
+---------------------------
+
+Minor Documentation changes, such as typo fixes, or addition / correction of a
+sentence require an accepting pull request review 
+by a project member, no rejecting reviews by a project member (lazy consensus). 
+Project members are expected to give “reasonable time” 
+to others to give their opinion on the pull
+request if they’re not confident others would agree. E-mails to the development
+mailing list are not required for minor documentation changes. 
+
+.. _code-change-process:
+
+Code Changes and Major Documentation Changes
+--------------------------------------------
+
+Code changes and major documentation changes in a pull request require approving reviews 
+by two project members,
+no rejecting reviews by a project member (lazy consensus). At any point in
+the pull request review process a reviewer may request that the author e-mail
+the development mailing list to continue the discussion outside of the pull
+request. An e-mail to yt-dev is not always required for changes of this
+magnitude, but it is an avenue that may be necessary to ensure that the change
+is in accordance with the project style and philosophy. If the code change is
+substantive enough, a member may request that the change be backed by a YTEP.  
+
+If a change of this magnitude 
+is proposed in an issue or the mailing list before a pull request is submitted, 
+the project recommends  
+that the issue receive positive/affirmitive feedback by at least one project
+member before a PR is submitted. The submitted PR will then go through the same
+requirements for merging as outlined in the first sentence of this section. 
+
+.. note:: 
+   While adding a new supporting frontend is generally a substantive contribution, 
+   it does not require a YTEP and falls under this category. 
```

Comment:
```suggestion
   it does not require a YTEP. 
```
The end of the sentence seems ambiguous to me, but I think the main message is just that no YTEP is required.